File: pgm/ABC_Dart/ABC_401_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  final int S = int.parse(stdin.readLineSync()!);
  final int twoHundred = 200;
  final int twoHundredNinetyNine = 299;
  final String Success = "Success";
  final String Failure = "Failure";
  print(twoHundred <= S && S <= twoHundredNinetyNine ? Success : Failure);
  exit(0);
}

File: pgm/ABC_PHP/ABC_401_A.php
<?php
$stream = fopen('php://stdin', 'r');
$line = fgets($stream);
$S = (int) trim($line);
fclose($stream);
const Success = 'Success';
const Failure = 'Failure';
const twoHundred = 200;
const twoHundredNinetyNine = 299;
echo (twoHundred <= $S && $S <= twoHundredNinetyNine
    ? Success
    : Failure) . PHP_EOL;
exit(0);
